src/sources/blank.ml

```diff
@@ -48,25 +48,28 @@ class blank ~kind duration =
         if remaining < 0 then Lazy.force Frame.size - position
         else min remaining (Lazy.force Frame.size - position)
       in
-      let content = ab.Frame.content in
       let video_pos = Frame.video_of_master position in
       (* Audio *)
-      Audio.clear
-        (Audio.sub content.Frame.audio
-           (Frame.audio_of_master position)
-           (Frame.audio_of_master length));
+      ( try
+          Audio.clear
+            (Audio.sub (AFrame.pcm ab)
+               (Frame.audio_of_master position)
+               (Frame.audio_of_master length))
+        with Frame_content.Invalid -> () );
```

Comment:
What is the purpose of this catched exception here?

Reply:
`Frame.pcm` requires `pcm` content type but the `Frame` may contain other type of contents. I decided to stick with only one type of exception coming out of `Frame_content` for now. I can add more but I'm not sure if that'll be useful.